### PR TITLE
Osd 4490 Service can delete EBS 

### PR DESCRIPTION
**Pull request by amolgautam25**
Service can delete EBS

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -106,6 +106,7 @@ func main() {
 				awsManager.CleanS3Instances(assumedRoleClient)
 				awsManager.CleanEc2Instances(assumedRoleClient)
 				awsManager.CleanUpAwsRoute53(assumedRoleClient)
+				awsManager.CleanVpcInstances(assumedRoleClient)
```
Comment (fahlmant) [inline main.go:109]: Is this supposed to be EBS?

Comment (amolgautam25) [reply main.go]: i have changed it.

**Comment by rogbas** [comment]
@drpaneas can you please review?

File: pkg/awsManager/ebsManager.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+		_, ebsSnapshotDeleteError := client.DeleteSnapshot(&ec2.DeleteSnapshotInput{SnapshotId: ebsSnapshotId})
+		if ebsSnapshotDeleteError != nil {
+			fmt.Println(ebsSnapshotDeleteError)
+			fmt.Print("Could not delete snapshot :", *ebsSnapshotId)
+			ebsSnapshotsNotDeleted = append(ebsSnapshotsNotDeleted, ebsSnapshotId)
+		}
+	}
+
+	if ebsSnapshotsNotDeleted != nil {
+		return errors.New("ERROR")
```
Comment (fahlmant) [inline pkg/awsManager/ebsManager.go:62]: Can this error be more descriptive?

File: pkg/awsManager/ebsManager.go [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+		_, err := client.DeleteVolume(&ec2.DeleteVolumeInput{VolumeId: ebsVolumeId})
+		if err != nil {
+			fmt.Println(err)
+			fmt.Print("Could not delete Volume :", *ebsVolumeId)
+			ebsVolumesNotDeleted = append(ebsVolumesNotDeleted, ebsVolumeId)
+		}
+	}
+
+	if ebsVolumesNotDeleted != nil {
+		return errors.New("ERROR")
```
Comment (fahlmant) [inline pkg/awsManager/ebsManager.go:113]: Same thing here

File: pkg/awsManager/ebsManager.go [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+	}
+
+	return nil
+}
+
+func CleanEbsSnapshots(client clientpkg.Client) error {
+	ebsSnapshotsToBeDeleted := ListEbsSnapshotForDeletion(client)
```
Comment (fahlmant) [inline pkg/awsManager/ebsManager.go:120]: Should these be separated? If an EBS fails to be deleted, could all the volumes still be deleted properly?

**Pull request merged**